targets/python/templates/APITest.py.ejs

```diff
@@ -0,0 +1,57 @@
+using PlayFab.<%- api.name %>Models;
```

Comment:
This is clearly C# code...
Convert it, or delete it.
For timeliness, we should probably use that blank "entry point" file above, make a couple of basic api calls and exit.  And delete this test file